components/invitation_modal/invite_as.tsx

```diff
@@ -49,24 +135,26 @@ export default function InviteAs(props: Props) {
                         },
                         {
                             key: (
-                                <span className='InviteAs__label'>
+                                <span className={`InviteAs__label ${guestDisabledClass}`}>
```

Comment:
classNames ?

Reply:
as personal preference, I rather to keep it simple using string literals if it is just one single conditional class.